ztp/gitops-subscriptions/argocd/example/image-based-upgrades/README.md

```diff
@@ -0,0 +1,91 @@
+## Image-Based Upgrades (IBU)
+This directory contains examples of generating resources required for Image Based Upgrades (IBU) utilizing the [Life Cycle Agent operator](https://github.com/openshift-kni/lifecycle-agent). These examples define policies to automate image-based upgrades, ensuring seamless deployment across managed clusters through Gitops.
+
+
+### Prerequisites
+
+Before using the IBU examples, ensure that the following namespaces have been created:
+
+- `ztp-common`: The root policy will be created in this namespace. If you use another name for the `common` namespace, please remember to update the `example-oadp-policy.yaml` policy.
+- `openshift-adp`: The ConfigMap containing the related OpenShift API for Data Protection (OADP) Custom Resources (CRs) will be copied to this namespace on the applicable spoke cluster(s).
+
+### Setup ArgoCD Application
+
+To deploy the IBU examples, you can use the existing [policies-app](https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/gitops-subscriptions/argocd/deployment/policies-app.yaml), which is also used for deploying DU profile policy examples. Refer to the [ReadMe](https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/gitops-subscriptions/argocd/README.md) section "Preparation of Hub cluster for ZTP" for detailed instructions on setting up the ArgoCD policies application.
+
+Ensure that your Git repository, which will be used with the ArgoCD policies application, contains a directory structured as follows:
+
+```plaintext
+├── source-crs/
+│   ├── ibu/
+│   │    ├── OpenShift.yaml
```

Comment:
Nit, I find the file name confusing. I guess the file is for backup restore CRs for platform artifacts? Can we have a name that indicates the nature of the CRs, like other source-cr files?

Reply:
Here are some alternative names:
1. OpenShiftBackupRestore.yaml
2. OpenShiftBR.yaml
3. OADPBackupRestore.yaml

Which do you find more appropriate? Feel free to suggest alternative options.

Reply:
How about `PlatformBackupRestore`?

Reply:
Sounds good to me! I will update the PR accordingly unless others have counter opinions (cc @browsell @imiller0)

Reply:
works for me